main/test_model.py

```diff
@@ -0,0 +1,41 @@
+# Lint as: python3
+"""Apply pre-trained model on test data"""
+
+from src import receive_data
+from src import suite
+import logging
+import pickle
+
+logging.basicConfig(
+    filename="main/test_model.log", filemode="w", level=logging.INFO)
+logging.basicConfig(level=logging.INFO)
+G_data = True
```

Comment:
I know you have this elsewhere, so maybe not for this pull request:
Rather than a flag that the user has to change in the code, maybe just test whether the data has an "id" column, and if so, rename if to "_id". (Or vice-versa; I can't tell which it's doing.)

Reply:
makes sense